assets/blocks/course-outline/course-outline-block.js

```diff
@@ -0,0 +1,32 @@
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+import { InnerBlocks } from '@wordpress/block-editor';
+
+registerBlockType( 'sensei-lms/course-outline', {
+	title: __( 'Outline', 'sensei-lms' ),
```

Comment:
I used the same titles that are in the design, but maybe should we rethink some of them as "Outline" can be very broad?

Reply:
Sorry for the delay. I think this should say "Course Outline", and probably we should put all Sensei blocks in their own "Sensei LMS" block category.

Reply:
Hey @donnapep ! Thank you for the feedback!

I updated it here https://github.com/Automattic/sensei/pull/3584/commits/9d51390b0e5ef52329d2fd1d8774d06cba4cf23e in this PR: https://github.com/Automattic/sensei/pull/3584

About the category, it's already implemented! 🎉